### PR TITLE
HARVESTER: Refactor combination keys of VNC

### PR DESCRIPTION
**Pull request by DaiYuzeng**
### Summary

- https://github.com/harvester/harvester/issues/379
- https://github.com/harvester/harvester/issues/1461

### CODE REVIEW

**Comment by DaiYuzeng** [comment]
Test Steps:
1. Create a VM with ubuntu desktop/windows
2. Complete ubuntu installation
3. Validate `Short Keys`
4. Click `Soft Reboot` and then the VM should be reboot

**Comment by DaiYuzeng** [comment]
Features Screenshot:

image

File: shell/components/novnc/NovncConsoleItem.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    addKeys({ key, pos }) {
+      this.$emit('update', { key, pos });
+    },
+
+    sendKeys() {
+      this.$emit('sendKeys');
+    },
+
+    getOpenStatus(key, pos) {
+      return this.path[pos] === key;
```
Comment (WuJun2016) [inline shell/components/novnc/NovncConsoleItem.vue:44]: We should not directly change the value passed in by props.

Comment (DaiYuzeng) [reply shell/components/novnc/NovncConsoleItem.vue:44]: @WuJun2016 

Func `getOpenStatus` only return comparison result. The prop won't be modified.

**Pull request merged**